### ISSUE TITLE
[suggestion] Smooth netherrack stairs

### ISSUE THREAD

**Issue by xkforce** [An-Sar/PrimalCore#297]
Everything else is there eg. slabs, chiseled blocks etc. but the stairs.

**Comment by An-Sar** [comment]
yeah a lot of stair variants are missing due to block ID limitations and i didn't want to use a tile entity. i could add stairs for everything and just have a config with them disabled by default like the wattle and daub...

**Comment by xkforce** [comment]
That would be a good solution.  Also you bring up a good point about block ids- currently primal registers 220.  Perhaps a toggle could be added to the config for groups of blocks like the one for stairs?  That could prune a few block ids too.

**Comment by An-Sar** [comment]
the existing stairs and ladder blocks actually have the config already, it's just at some point i stopped adding new sub blocks when i got preoccupied by block count. they just currently default to enabled, unlike daub bc it's a massive consumer of flat blocks.
